spk/rclone/Makefile

```
@@ -0,0 +1,25 @@
+SPK_NAME = rclone
+SPK_VERS = 1.69.1
+SPK_REV = 1
+SPK_ICON = src/logo.png
+
+DEPENDS = cross/$(SPK_NAME)
+
+UNSUPPORTED_ARCHS = $(PPC_ARCHS)
+
+MAINTAINER = dotwee
+DESCRIPTION = Rclone is a command-line program to manage files on cloud storage. It is a feature-rich alternative to cloud vendors' web storage interfaces.
+DISPLAY_NAME = rclone
+STARTABLE = no
+
+CHANGELOG = "Initial release."
+
+HOMEPAGE = https://rclone.org/
+HELPURL = https://rclone.org/docs/
+
+LICENSE  = MIT License
+LICENSE_FILE = $(WORK_DIR)/$(SPK_NAME)-$(SPK_VERS)/COPYING
```

since it is MIT license, I would like to avoid the license confirmation in the install/upgrade wizard
```suggestion
LICENSE = MIT
```